src/objects/Hierarchy_Settings__c.object

```diff
@@ -15,6 +15,28 @@
         <unique>false</unique>
     </fields>
     <fields>
+        <fullName>Affl_ProgEnroll_Del__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>Determines is an Affiliation will be automatically deleted when the related Program Enrollment is deleted. Otherwise the status will be changed to Former.</description>
```

Comment:
If "former" the default value? Or is it pulling from something else? 

Also, "Determined if" not "Determines is"